Buildings/Applications/DataCenters/ChillerCooled/Examples/BaseClasses/PartialDataCenter.mo

```diff
@@ -246,11 +249,11 @@ partial model PartialDataCenter
     controllerType=Modelica.Blocks.Types.SimpleController.PI,
     Ti=40,
     yMin=0.2,
-    k=0.001)
+    k=0.0001)
```

Comment:
It would be better if you divide the measured pressure by dpSetPoi (using a gain), and then use for `pumSpe.u_s` a value of `1`. This allows you to use a more reasonable gain, and it will allow tuning the controller independent of the units of measurement for pressure, which can vary orders of magnitude depending on the unit that is used.